spec/factories/goods_nomenclature_factory.rb

```diff
@@ -236,6 +236,10 @@
       validity_end_date   { 1.year.ago.beginning_of_day  }
     end
 
+    trait :with_heading do
```

Comment:
Am I missing something - this feels like it would confuse a future developer?

If I call `factory :goods_nomenclature, :with_heading` - I'd anticipate it creating me a goods nomenclature with a heading? But this doesn't look like it will?

Reply:
Given only specific types of goods nomenclature have headings I think this okay. Updated the comment to reflect this